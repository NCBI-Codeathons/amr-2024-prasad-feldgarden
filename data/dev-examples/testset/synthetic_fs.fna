>ompK35_C300CC  AJ011501.1:123-1202 manually mutated with insertion after position 300
ATGATGAAGCGCAATATTCTGGCAGTGGTGATCCCTGCCCTGCTGGTAGC
CGGTGCAGCCAACGCTGCAGAAATCTATAACAAAAACGGCAACAAACTGG
ACTTCTATGGAAAAATGGTCGGCGAGCACGTCTGGACCACCAATGGCGAC
ACCAGCAGCGACGATACCACCTATGCCCGTATCGGCCTGAAAGGCGAAAC
TCAGATCAACGATCAGCTGATCGGCTACGGCCAGTGGGAATACAACATGG
ACGCGTCCAATGTTGAAGGTTCCCAGACCACAAAAACCCGTCTGGCGTTCC
GCAGGCCTGAAAGCGGGCGAATACGGTTCATTCGACTATGGCCGTAACTA
CGGCGCGATCTACGACGTCGAAGCGGCAACCGATATGCTGGTTGAATGGG
GCGGTGACGGCTGGAACTACACCGACAACTACATGACCGGTCGTACCAAC
GGCGTCGCAACCTACCGTAACTCTGACTTCTTCGGTCTGGTTGACGGTCT
GAGCTTCGCGCTGCAGTACCAGGGTAAAAACGACCACGACCGTGCGATTC
GCAAGCAGAATGGCGACGGCTTCAGCACCGCAGCCACCTACGCGTTCGAC
AACGGTATCGCACTGTCTGCAGGCTACTCCAGCTCTAACCGTAGCGTCGA
TCAGAAAGCTGACGGCAATGGCGACAAAGCCGAAGCCTGGGCGACCTCTG
CAAAATATGACGCTAACAACATCTATGCGGCCGTCATGTACTCCCAGACT
TACAACATGACTCCGGAAGAAGATAACCACTTCGCCGGTAAAACTCAGAA
CTTTGAAGCAGTTGTACAGTATCAGTTTGACTTCGGCCTGCGTCCGTCCA
TCGGCTACGTACAGACCAAAGGCAAGGACCTGCAGTCGCGTGCTGGCTTC
TCCGGCGGCGATGCGGATCTGGTTAAATACATCGAAGTGGGTACCTGGTA
CTACTTTAACAAGAACATGAACGTCTACGCTGCGTATAAATTCAACCAGC
TGGACGACAACGATTACACCAAAGCGGCTGGTGTCGCCACTGACGACCAG
GCGGCCGTGGGTATCGTTTACCAATTCTAA
