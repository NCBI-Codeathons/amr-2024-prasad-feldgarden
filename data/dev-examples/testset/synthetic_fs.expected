element_symbol	contig_acc	start	stop	orientation	lesion_type	aa_identity	nt_identity	ref_accession	lesion_location
OmpK35	ompK35_C300CC	1	1081	+	FRAMESHIFT	100	99	AJ011501.1	101
